README.md

```diff
@@ -773,6 +773,14 @@ Sniff provides the following settings:
 
 However, if you prefer Yoda conditions, you can use `RequireYodaComparisonSniff`.
 
+#### SlevomatCodingStandard.Files.FileLineCount
```

Comment:
Is there a reason why not to be consistent and name it `FileLength` ?

Reply:
My reasoning was to further "disambiguate" from Files/LineLengthSniff 

LineLengthSniff   // sniffs # of columns in line  (not number of lines)
FileLengthSniff   // sniffs # of lines in file    ( feel like  "LineCountSniff"  is more descriptive  ?)

Reply:
However have FunctionLengthSniff and it checks number of liness.

Reply:
👍 / will rename

Reply:
done